app/api/helpers/files.py

```diff
@@ -78,13 +79,17 @@ def create_save_resized_image(image_file, basewidth=None, maintain_aspect=None,
         return None
 
     filename = '{filename}.{ext}'.format(filename=get_file_name(), ext=ext)
-    data = urllib.request.urlopen(image_file).read()
+    try:
+        data = urllib.request.urlopen(image_file).read()
+    except (urllib.error.HTTPError, urllib.error.URLError) as e:
+        raise UnprocessableEntity(
+            {'source': 'attributes/original-image-url'}, 'Invalid Image URL'
```

Comment:
Even though we are using it from only one position, if we want to have such specific error messages then add this method in that class only. Adding it in a helper class means we can't use it elsewhere in the project. we may want to use it in the future from a different location as well. Let's follow the best practices.